src/player.cpp

```diff
@@ -13503,8 +13503,7 @@ void player::print_encumbrance( WINDOW *win, int line, item *selected_clothing )
     for( auto bp : parts ) {
         const encumbrance_data &e = enc_data[bp];
         bool highlighted = ( selected_clothing == nullptr ) ? false :
-            ( selected_clothing->covers( static_cast<body_part>( bp ) ) ||
-              selected_clothing->covers( static_cast<body_part>( bp_aiOther[bp] ) ) );
+            ( selected_clothing->covers( static_cast<body_part>( bp ) ) );
```

Comment:
The problem is a little more complex than that. The `covers` set tells no difference between "either limb" and "both limbs" (legs, arms, whatever). With the second line removed, you'll get only one foot highlighted for boots, one hand for gloves, etc. Use `item->is_sided()` and `item->get_side()` to properly distinguish.